code/modules/spells/spell_types/godhand.dm

```diff
@@ -182,3 +182,38 @@
 
 	if(charges <= 0)
 		qdel(src)
+
+/obj/item/melee/touch_attack/nuclearfist
+	name = "\improper PURE MANLINESS"
+	desc = "SHOW THEM RAW POWER"
+	catchphrase = "I CAST FIST!"
+	on_use_sound = 'sound/weapons/nuclear_fist.ogg'
+	icon_state = "disintegrate"
+	item_state = "disintegrate"
+
+/obj/item/melee/touch_attack/nuclearfist/afterattack(atom/movable/target, mob/living/carbon/user, proximity)
+	if(!proximity || target == user || !ismob(target) || !iscarbon(user) || user.lying || user.handcuffed) //exploding after touching yourself would be bad
+		return
+	if(!user.can_speak_vocal())
+		to_chat(user, "<span class='notice'>You can't get the words out!</span>")
+		return
+	var/mob/M = target
+	var/atom/A = M.anti_magic_check()
```

Comment:
Something something holy protection, and tinfoil hats will stop this.

Reply:
You think I should forego having this spell be blocked by magic immunities?

Reply:
for reference, no, tinfoil hats will not stop this, because the arguments for anti_magic_check are magic = TRUE, holy = FALSE, tinfoil = FALSE, chargecost = 1, self = FALSE by default and tinfoil hats are magic = FALSE, holy = FALSE, tinfoil = TRUE